manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "description": "Hong Kong Open Source Conference is a annual international open source conference in Hong Kong. It is aimed to demonstrate the use of open source software technology, open hardware technology and latest development in the world.",
+  "developer": {
+    "name": "HKOSCon 2017",
+    "url": "http://hkoscon.org/2017/"
+  },
+  "icons": {
+    "48": "favicon.png"
+  },
+  "manifest_version": 2,
+  "name": "Hong Kong Open Source Conference 2017",
+  "short_name": "HKOSCon 2017",
+  "web_accessible_resources": [
```

Comment:
Wrong file names, please use those generated files in public by `gulp build` and add prefix `/2017`